packages/flutter_markdown/lib/src/markdown.dart

```diff
@@ -0,0 +1,472 @@
+part of flutter_markdown;
+
+class MarkdownStyle {
+
+  MarkdownStyle({
+    this.a,
+    this.p,
+    this.code,
+    this.h1,
+    this.h2,
+    this.h3,
+    this.h4,
+    this.h5,
+    this.h6,
+    this.em,
+    this.strong,
+    this.blockquote,
+    this.blockSpacing,
+    this.listIndent,
+    this.blockquotePadding,
+    this.blockquoteDecoration,
+    this.codeblockPadding,
+    this.codeblockDecoration
+  }) {
+    _init();
+  }
+
+  MarkdownStyle.defaultFromTheme(ThemeData theme) :
+    a = new TextStyle(color: Colors.blue[500]),
+    p = theme.text.body1,
+    code = new TextStyle(
+      color: Colors.grey[700],
+      fontFamily: "monospace",
+      fontSize: theme.text.body1.fontSize * 0.85
+    ),
+    h1 = theme.text.headline,
+    h2 = theme.text.title,
+    h3 = theme.text.subhead,
+    h4 = theme.text.body2,
+    h5 = theme.text.body2,
+    h6 = theme.text.body2,
+    em = new TextStyle(fontStyle: FontStyle.italic),
+    strong = new TextStyle(fontWeight: FontWeight.bold),
+    blockquote = theme.text.body1,
+    blockSpacing = 8.0,
+    listIndent = 32.0,
+    blockquotePadding = 8.0,
+    blockquoteDecoration = new BoxDecoration(
+      backgroundColor: Colors.blue[100],
+      borderRadius: 2.0
+    ),
+    codeblockPadding = 8.0,
+    codeblockDecoration = new BoxDecoration(
+      backgroundColor: Colors.grey[100],
+      borderRadius: 2.0
+    ) {
+    _init();
+  }
+
+  MarkdownStyle.largeFromTheme(ThemeData theme) :
+    a = new TextStyle(color: Colors.blue[500]),
```

Comment:
IMHO, markdown shouldn't depend on material.dart.  Instead, you should create your own color constants.